### ISSUE TITLE
Upgrade a module and then use it again (incorrect content)

### ISSUE THREAD

**Issue by yapper-git** [mjdenham/and-bible#91]
It seems Andbible does not reload the module after an upgrade.

Indeed I upgraded the module FreJND from 1.21.1 to 1.22 and returned to reading window, but the display was buggy (see screenshot below, tags mixed with the text). I tried to change bible module and loop to FreJND again, but it didn't work. I needed to restart the application (close in application list, and restart).

![attached screenshot](https://cloud.githubusercontent.com/assets/9322299/26172017/30954fc0-3b47-11e7-89dc-a276e76346ab.png)

**Comment by tuomas2** [comment]
Also this
https://support.andbible.org/scp/tickets.php?id=1330

**Comment by tuomas2** [comment]
Still with version 4.0.640

**Comment by timbze** [comment]
I don't think this is an issue anymore. I just updated a module and it did update right away.
